### PR TITLE
Script to run automated Vale review on added/modified content in PR

### PR DESCRIPTION
**Pull request by rohennes**
Adding script to run automated Vale review on changed/modified content in a PR. For integration with Prow.

### CODE REVIEW

File: test.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Don't do this.
```
Comment (rohennes) [inline test.adoc:1]: Module is missing the '_mod-docs-content-type' variable.

File: scripts/vale-review.sh [hunk 2]
```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# This runs Vale on updated files, checks if the Vale alerts are on new/modified lines, and if so, builds curl request for GitHub review comment API
+
```
Comment (aireilly) [inline scripts/vale-review.sh:3]: Not strictly required tbh. but useful for anyone who wants to poke at this in future.

```suggestion
# Check if jq is installed
hash jq 2>/dev/null || { echo >&2 "Error: jq is not installed"; exit 1; }
```

File: scripts/vale-review.sh [hunk 3]
```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# This runs Vale on updated files, checks if the Vale alerts are on new/modified lines, and if so, builds curl request for GitHub review comment API
```
Comment (aireilly) [inline scripts/vale-review.sh:2]: Cna we also add a line that in order to test this locally, you must `export $GITHUB_AUTH_TOKEN`

File: scripts/vale-review.sh [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                fi
+            fi
+                
+        done <<< "$FULL_DIFF"
+     
+    done
+
+done
+
+rm vale_output_temp.json
```
Comment (aireilly) [inline scripts/vale-review.sh:94]: nit - this isn't required in build env and anyone who runs this locally will want to see output

**Pull request merged**